HEN_HOUSE/omega/beamnrc/beamnrc_user_macros.mortran

```diff
@@ -86,8 +86,8 @@ REPLACE {$COVMIN} WITH {10}        "min. no. of particles crossing a scoring"
                                    "average energy/angle only)"
 REPLACE {$MXMED} WITH {12}         "max # of media
 REPLACE {$MXREG} WITH {3000}        "maximum number of regions (absolute)
-REPLACE {$MXSTACK} WITH {10000}    "maximum stack (much less if no brem split)
-REPLACE {$MAXBRSPLIT} WITH {2000}  "maximum number of bremsstrahlung splits
+REPLACE {$MXSTACK} WITH {1000000}    "maximum stack (much less if no brem split)
+REPLACE {$MAXBRSPLIT} WITH {500000}  "maximum number of bremsstrahlung splits
```

Comment:
@blakewalters can you confirm that committing these increases in default sizes is intentional?

Reply:
Absolutely unintentional @ftessier; these are the settings I routinely use.  Thanks for catching this!  I'll return them to their original defaults.

Reply:
All right, just push the change as a new commit on top, as discussed!